core/trino-main/src/test/java/io/trino/block/BlockAssertions.java

```diff
@@ -326,19 +326,26 @@ public static Block createRandomStringBlock(int positionCount, float nullRate, i
     private static Block createRandomVarbinariesBlock(int positionCount, float nullRate)
     {
         Random random = random();
-        return createSlicesBlock(VARBINARY, generateListWithNulls(positionCount, nullRate, () -> Slices.wrappedLongArray(random.nextLong(), random.nextLong())));
+        return createSlicesBlock(VARBINARY, generateListWithNulls(positionCount, nullRate, () -> randomSlice(random, 16)));
     }
 
     private static Block createRandomUUIDsBlock(int positionCount, float nullRate)
     {
         Random random = random();
-        return createSlicesBlock(UUID, generateListWithNulls(positionCount, nullRate, () -> Slices.wrappedLongArray(random.nextLong(), random.nextLong())));
+        return createSlicesBlock(UUID, generateListWithNulls(positionCount, nullRate, () -> randomSlice(random, 16)));
     }
 
     private static Block createRandomIpAddressesBlock(int positionCount, float nullRate)
     {
         Random random = random();
-        return createSlicesBlock(IPADDRESS, generateListWithNulls(positionCount, nullRate, () -> Slices.wrappedLongArray(random.nextLong(), random.nextLong())));
+        return createSlicesBlock(IPADDRESS, generateListWithNulls(positionCount, nullRate, () -> randomSlice(random, 16)));
+    }
+
+    private static Slice randomSlice(Random random, int length)
```

Comment:
Should we add this to Slice?

Reply:
We could add this to Slices.  I tooks a quick look at the code base, and there are like 20ish places that do this.